### PR TITLE
added config parsing script

### PR DESCRIPTION
**Pull request by adamdevigili**

### CODE REVIEW

**Comment by pwplusnick** [comment]
@adamdevigili I like it, but I think we are trying to keep the scripts as deploy agnostic as possible. If you can generalize it a little bit, we would be more than willing to take this! Thanks.

@CullenTaylor Could you also check it out.

**Comment by epwn** [comment]
-1 i concur it should be as "deploy agnostic" as possible.

File: depl/pre/config-parser.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    else:
+        break
+
+out_file = open('config', 'w')
+
+for line in in_file:
+    outline = ""
+    if line.strip():
+        if "100-node" in line:
+            name = line[9:-24]
```
Comment (davidcusatis) [inline depl/pre/config-parser.py:20]: magic numbers???????????????????????? breh

File: depl/pre/config-parser.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            if 'kvm' in name:
+                name = name[3:]
+                name = "compute" + name
+
+            outline = outline + "Host " + name + '\n'
+
+        if "10." in line and '100-node' not in line:
+            outline = outline + '  HostName ' + line + '  User root\n  IdentityFile ~/.ssh/id_rsa\n\n'
```
Comment (davidcusatis) [inline depl/pre/config-parser.py:28]: not pep8 :)

File: depl/pre/config-parser.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
+#Use this script to take copy and pasted server information from SoftLayer's
```
Comment (davidcusatis) [inline depl/pre/config-parser.py:1]: not pep8, comments need a space after #

**Pull request closed without merging**